pkg/segment/query/processor/evalcommand.go

```diff
@@ -18,22 +18,130 @@
 package processor
 
 import (
+	"fmt"
+	"io"
+
 	"github.com/siglens/siglens/pkg/segment/query/iqr"
 	"github.com/siglens/siglens/pkg/segment/structs"
+	"github.com/siglens/siglens/pkg/segment/utils"
+	toputils "github.com/siglens/siglens/pkg/utils"
 )
 
 type evalProcessor struct {
 	options *structs.EvalExpr
 }
 
+func evaluateValueExpr(valueExpr *structs.ValueExpr, fieldToValue map[string]utils.CValueEnclosure) (interface{}, error) {
```

Comment:
Can we use the existing `ValueExpr.EvaluateValueExpr()` method instead of making this? If not, would it make sense to make this function a method on `ValueExpr` and move it to that file?